controllers/etcd_controller.go

```diff
@@ -1544,29 +1439,3 @@ func (r *EtcdReconciler) claimConfigMaps(ctx context.Context, etcd *druidv1alpha
 	cm := NewEtcdDruidRefManager(r.Client, r.Scheme, etcd, selector, etcdGVK, canAdoptFunc)
```

Comment:
This comment refers to line #1425:
The function `claimConfigMaps` is unused. Can you remove this?

Reply:
Removed